test/e2e/e2e-test/src/test/resources/expected/gateway/instances.yml

```diff
@@ -16,12 +16,3 @@
 instances:
   - key: not null
     label: not null
-    attributes:
-      - name: OS Name
-        value: not null
-      - name: hostname
-        value: not null
-      - name: Process No.
-        value: gt 0
-      - name: ipv4s
-        value: not null
```

Comment:
Why remove all attributes checking? This is passed, and you only add new attributes rather than deleting.

Reply:
How to deal with the non java client without newly added attributes?

Reply:
These attributes are new, no all language agent reports this. Existing e2e tests have their own logic. I can't say they are correct, but directly removing clearly is wrong.